sent.py

```diff
@@ -382,10 +381,22 @@ def fetch_contract_statuses(signum):
 def fetch_service_nodes(signum):
     app.logger.warning("{} Update Service Nodes Start".format(date_now_str()))
     omq, oxend            = omq_connection()
-    app.nodes             = get_sns_future(omq, oxend).get()["service_node_states"]
+    nodes                 = get_sns_future(omq, oxend).get()["service_node_states"]
+    # Reset state and re-populate it
+    app.sn_map = {}
     app.node_contributors = {}
 
-    for index, node in enumerate(app.nodes):
+    [ids, bls_keys] = app.service_node_rewards.node_ids()
+    formatted_bls_keys = {f"{x:064x}{y:064x}": node_id for node_id, (x, y) in zip(ids, bls_keys)}
+
+    for index, node in enumerate(nodes):
+        node["node_id"] = formatted_bls_keys.get(node["pubkey_bls"])
```

Comment:
In the codebases we're been calling `node_id` `contract_id` if you could update that to make it consistent.